dev-python/sqlglot/sqlglot-17.8.4.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} )
+
+inherit distutils-r1 pypi optfeature
+
+DESCRIPTION="An easily customizable SQL parser and transpiler"
+HOMEPAGE="
+	https://sqlglot.com
+	https://github.com/tobymao/sqlglot/
+	https://pypi.org/project/sqlglot/
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+#BDEPEND="
+#	test? (
+#		>=dev-python/duckdb-0.6[${PYTHON_USEDEP}]
+#		dev-python/pandas[${PYTHON_USEDEP}]
+#		dev-python/pyspark[${PYTHON_USEDEP}]
+#	)
+#"
```

Comment:
Let's not keep deps that we don't actually use. If someone has to actually enable them in the future, they need to recheck the package anyway, and this could lead to stale deps being used.